.travis.yml

```diff
@@ -36,9 +36,9 @@ matrix:
     compiler: ": #GHC 8.8.4"
     addons: {apt: {packages: [cabal-install-2.4,ghc-8.8.4], sources: [hvr-ghc]}}
 
-  - env: BUILD=cabal GHCVER=8.10.2 CABALVER=2.4
-    compiler: ": #GHC 8.10.2"
-    addons: {apt: {packages: [cabal-install-2.4,ghc-8.10.2], sources: [hvr-ghc]}}
+  - env: BUILD=cabal GHCVER=8.10.3 CABALVER=2.4
```

Comment:
Probably the whole file itself can be removed if we are migration away from Travis. I see that you mentioned that github action doesn't seem to run. Is it because of not having something like this:

```
on:
    pull_request:
    push:
      branches:
      - master
```

Reply:
I think the reason it doesn't run is because github is set up so that if you introduce CI stuff in a PR, it has to be accepted before it starts doing stuff. 
```yaml
  pull_request:
    branches: '*'
```
makes it so that a PR to any branch triggers the CI, as you can see [here](https://github.com/iyefrat/libmpd-haskell/actions/runs/507142357). I just left the travis in temporarily so you can see that the CI of all the changes passes on this PR, I'll remove it now.